### ISSUE TITLE
Website Accessibility-WCAG 2.4.4,WCAG 4.1.2: Ensures links have discernible text (.w-inline-block:nth-child(3))

### ISSUE THREAD

**Issue by A11Ybot-OP** [matrix-org/matrix.org#1211]
#### Issue
`Ensures links have discernible text` ([link-name](https://accessibilityinsights.io/info-examples/web/link-name))
#### Target application
[Matrix.org](https://matrix.org/)
#### Element path
.mxfooter__iconwrapper > .w-inline-block:nth-child(3)
#### Snippet
`<a href="https://www.youtube.com/channel/UCVFkW-chclhuyYRbmmfwt6w" class="w-inline-block"><img src="/images/youtube.svg" alt="" class="mxfooter__icon"></a>`
#### How to fix
    Fix all of the following:
      Element is in tab order and does not have accessible text
    
    Fix any of the following:
      Element does not have text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
#### Environment
Microsoft Edge version 96.0.1054.62
